core/controllers/classifier_test.py

```diff
@@ -67,9 +65,18 @@ def setUp(self):
             '_c_lw': [],
             '_c_l': []
         }
-        self.job_id = classifier_services.create_classifier_training_job(
-            algorithm_id, interaction_id, self.exp_id, self.exploration.version,
-            'Home', training_data, feconf.TRAINING_JOB_STATUS_PENDING)
+        classifier_training_job = (
+            classifier_services.get_classifier_training_job(
+                self.exp_id, self.exploration.version, 'Home'))
+        self.job_id = classifier_training_job.job_id
+        # Use the mark_training_job_pending function below after this is merged
```

Comment:
OK. I think it's a good idea to expand on the comment a bit more, then, with this info, explaining exactly which lines should be replaced by what. Also, please put a `TODO(pranavsid98): ` before it. That helps make clear that it's a TODO and describes exactly what needs to be done.